domehunter/gRPC-server/hx2dome.proto_server.py

```diff
@@ -52,11 +54,11 @@ class HX2DomeServer(hx2dome_pb2_grpc.HX2DomeServicer):
 
     """
 
-    def __init__(self, testing, debug_lights, server_testing):
+    def __init__(self, **kwargs):
```

Comment:
`Dome` has `home_azimuth` as a positional argument.

```suggestion
    def __init__(self, *args, **kwargs):
```

Reply:
I was doing that but I was passing in as an arg but didn't remove it from kwargs as well so it was getting two assignments of `home_azimuth`. Proper thing to do would be just remove it from kwargs and only pass in via args. I just noticed that it still works if I only pass in kwargs so left it like that.